SU2_CFD/src/solver_structure.cpp

```diff
@@ -226,7 +226,8 @@ void CSolver::SetResidual_RMS(CGeometry *geometry, CConfig *config) {
       
     }
     
-    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/Global_nPointDomain)));
+    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/ geometry->GetnPoint() )));
```

Comment:
You know better than I which value to use there. But in serial the result is wrong - that's my only point. It should be correct and the same in serial and in parallel.